R/pkg/NAMESPACE

```diff
@@ -10,6 +10,7 @@ export("sparkR.session")
 export("sparkR.init")
 export("sparkR.stop")
 export("sparkR.session.stop")
+export("conf")
```

Comment:
also this is readonly - `setter` is `spark.session()` - in line with SparkSession is builder syntax.